compile error on armv7 for your in-kernel registry

Luca Weiss: compile error on armv7 for your in-kernel registry
```
drivers/soc/qcom/qcom_sns_reg.c: In function 'qcom_sns_reg_get_group_req_handler':
drivers/soc/qcom/qcom_sns_reg.c:296:15: error: implicit declaration of function 'kzalloc' [-Wimplicit-function-declaration]
  296 |         rsp = kzalloc(sizeof(*rsp), GFP_KERNEL);
      |               ^~~~~~~
```
seems to need #include <linux/slab.h>

--- a/drivers/soc/qcom/qcom_sns_reg.c
+++ b/drivers/soc/qcom/qcom_sns_reg.c
@@ -15,6 +15,7 @@
 #include <linux/of.h>
 #include <linux/refcount.h>
 #include <linux/remoteproc/qcom_rproc.h>
+#include <linux/slab.h>
 #include <linux/soc/qcom/qmi.h>
 #include <linux/workqueue.h>
 

